kube-logging-logging-operator.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: kube-logging-logging-operator
+  version: 4.2.2
+  epoch: 0
+  description: Logging operator for Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kube-logging/logging-operator
+      tag: ${{package.version}}
+      expected-commit: 469986c420204c2963a87df9a286c751ec6d77ec
+
+  - runs: |
+      CGO_ENABLED=0 GO111MODULE=on go build -o bin/manager main.go
+      mkdir -p ${{targets.destdir}}/usr/bin
+      install -Dm755 ./bin/manager ${{targets.destdir}}/usr/bin/manager
+
+  - uses: strip
+
+subpackages:
+  - name: "kube-logging-logging-operator-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          # The helm chart expects the logging-operator binaries to be in / instead of /usr/bin
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+      - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: kube-logging/logging-operator
+    tag-filter: 4.2.2
```

Comment:
This won't match any other tags so this won't update automatically

Reply:
I fixed it, thank you.